PPP.py

```diff
@@ -265,8 +282,16 @@ def setupVariables():
             local_convert = 'u2w'
 
     playlist = [convertPath(track, local_convert, False) for track in playlist]
+    
 
-    local_prepend = os.path.commonpath(playlist)
```

Comment:
Could you double check the original code on your setup? `convertPath` above this should convert the playlist variable to the same path format as your PPP machine, so converting further down shouldn't be necessary? The line after should then convert back to it's original format for future use